spec/models/api_google_docs_search_spec.rb

```diff
@@ -174,6 +178,10 @@
     end
 
     context 'when Google response contains empty results' do
+
+      before { I18n.locale = :es }
```

Comment:
@HalahAb , nothing in this test seems to require that the language be set to `es`...? I would expect that the only time these `before` and `after` lines are required would be for specs with a context like `when the site locale is es`.

Reply:
I think it was causing new cassettes files to be generated, because when ran in order the previous example didn't switch back to en and the original cassette was in es for this context. I think I was trying to avoid that, but I've added the modified cassettes now.